static/js/runtime-config-listeners/custom-autocomplete-prompts.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @type {import('../runtime-config.js').RuntimeConfigListener}
+ */
+ export default {
+  key: 'customPrompts',
+  callback: value => {
+    ANSWERS.components._activeComponents
```

Comment:
I think we'd rather not reach into ANSWERS.components if possible

Reply:
I'm not sure of another way of getting the autocomplete instances without going through ANSWERS.components at the moment. The exposed function `getActiveComponent` from ANSWERS.components only provide one instance so I had to use the internal field `_activeComponents` to work. I could add another method `getActiveComponents` to ANSWERS.component or to top level ANSWERS to avoid this, thoughts on that?

Reply:
instead of directly calling methods on the SearchBar's autocomplete, could we add config to the SearchBar instead?

Reply:
after talking with yen, trying to have some config callback would probably be more trouble than its worth